src/apps/reports/src/app/views/Metrics/index.js

```diff
@@ -58,12 +58,12 @@ const numberWithCommas = (x) => {
 
 const floatWithCommas = (x) => {
   x = x.toFixed(2);
-  let parts = x.toString().split(".");
+  const parts = x.toString().split(".");
```

Comment:
No changed needed but want to provide feedback for future reference. With an error fix it is best to avoid making additional changes which are not directly related to the error at hand. In this case the change from `let` to `const`. Can make if difficult to pin point the exact change which solves the issue. Obviously with this one it's not all that difficult but it's a good habit to follow so when there are errors that require cross cutting changes the PR is easier to understand knowing that every change is necessary. 

Additionally in terms of the change from let to const this particular line and it's following line is a good example of the debate on where it is appropriate to use const in javascript. While the assignment of `parts` is not changed on line 62 the underlying value is. i.e. The array the const is referring to has its values modified. While technically okay, like the other point, it is best to avoid making a habit of describing structures which have their values modified with `const`.